Vagrantfile

```diff
@@ -9,7 +9,9 @@ Vagrant.configure(2) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
   config.vm.box = "ubuntu/trusty64"
-
+  config.vm.synced_folder "./", "/vagrant",
```

Comment:
how does this solves the problem?
I am confused,
is this like the default setup?